test/integration/smoke-test.js

```diff
@@ -15,9 +15,22 @@ describe('smoke', () => {
       .reply(200, {})
 
     const github = new GitHub({
-      host: 'myhost.com',
+      baseUrl: 'http://myhost.com/my/api'
+    })
+
+    return github.orgs.get({org: 'myorg'})
+  })
+
+  it('protocol, port, pathPrefix coverage', () => {
+    nock('http://myhost.com')
+      .get('/my/api/orgs/myorg')
+      .reply(200, {})
+
+    const github = new GitHub({
       protocol: 'http',
-      pathPrefix: '/my/api/'
+      host: 'myhost.com',
+      pathPrefix: '/my/api',
+      baseUrl: null
     })
```

Comment:
remove the `protocol, port, pathPrefix coverage` test here, we should cover it all in `deprecations-test.js`